proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java

```diff
@@ -13,10 +13,13 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.Set;
+import java.util.regex.Pattern;
+
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 public class VelocityCommandManager implements CommandManager {
 
+  private static final Pattern SPLIT_PATTERN = Pattern.compile(" ");
```

Comment:
Revert this. String#split is faster. 
https://stackoverflow.com/a/14901754
https://github.com/SpigotMC/BungeeCord/pull/2539

Reply:
Guava's `Splitter` works too.
https://google.github.io/guava/releases/23.0/api/docs/com/google/common/base/Splitter.html

Reply:
> 
> 
> Guava's `Splitter` works well too.
> https://google.github.io/guava/releases/23.0/api/docs/com/google/common/base/Splitter.html

Gonna make a quick JMH benchmark.

Reply:
> 
> 
> > Guava's `Splitter` works well too.
> > https://google.github.io/guava/releases/23.0/api/docs/com/google/common/base/Splitter.html
> 
> Gonna make a quick JMH benchmark.

Yeah, Guava's `Splitter` is much faster then Java's one.